subjects/prime_checker/README.md

```diff
@@ -4,9 +4,11 @@
 
 Create a **function** `prime_checker` that takes an `u32` and check if it is a prime number.
 
-The result will be `None` if the argument is less or equal one otherwise, it will return the result with the prime number inside if the number is prime, or it will return an `enum` `PrimeErr` if the `u32` is not prime.
+The result will be `None` if the argument is less or equal one otherwise, it will return a `Result`.
+If the `u32` is prime, the function will return an`Ok(u32)`; for any other case it will return an `enum` `PrimeErr`.
```

Comment:
- Missing a space for "an`Ok(u32)`"
- To use ";" seems very strange to me, I suggest to use "." or "," as ";" is nowadays almost never used.